src/lib.js

```diff
@@ -194,9 +200,14 @@ export function filterStale(projects) {
   return filtered
 }
 
-export function filterNew(projects, numberOfDaysBack){
+export function filterNew(projects){
   // projects is an array of objects; numberOfDaysBack is the number of days to search back from day of email
```

Comment:
let's edit this comment, since 'numberOfDaysBack' is removed

Reply:
Oh good catch, thanks